openshift/oadp-operator
# fix: clean up code around velero server flags CLI command

## Description

This PR should be fix duplication, missing fields and wrong descriptions around velero server flags code.

## What was done

I was checking for wrong descriptions in our CRDs and then came across the velero server flags code. I tried to remove duplication and add TODOs around missing fields. As discussed, that code should be automated: each time velero is updated in `go.mod`
- a check should be run to check if all fields from `velero help server` are present in DPA (maybe skip some, but add why in code as comments) 
- and if their descriptions are correct (with some auto corrections).

## How to test

Check if the removal of duplication around velero server flags code did not change the behavior of the args passed to velero container.

## TODO

This PR SHOULD NOT be merged. It does to many things. Break up in smaller ones.
